src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +14,10 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
     public boolean login(String email, String password) {
-        return false;
+        User user = userService.findByEmail(email);
+        return user != null && email.equals(user.getEmail())
```

Comment:
There is no reason to compare emain because email checking logic already defined in UserService#findByEmail, so we can just compare passwords